docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,141 @@
+Suggesting Apps for Split Tunneling
+===================================
+
+Epic: [VPN-4412: Suggest Apps to Exclude from VPN Protection](https://mozilla-hub.atlassian.net/browse/VPN-4412)
+
+Context / Problem Statement
+---------------------------
+
+We're looking to create tools that prompt the user in ways that help them stay more secure. Split tunneling (called 'App Exclusion' to users) is a feature that can reduce the need to turn off the VPN entirely by excluding traffic in specific apps from the VPN tunnel, but many users don't know about it. The current feature is in settings, and provides an alphabetical checklist of installed apps:
+
+<img src="./images/0004-01.png" height=500>
+
+This epic is for a new feature to proactively let users know when they have apps that could benefit from the split tunneling feature.
+
+Assumptions & Out-of-scope
+--------------------------
+
+-   This is only for platforms that currently support split tunnel - Windows, Linux, and Android. We will not build split tunneling into macOS or iOS as part of this work.
+-   Notifications refer to system-level notifications. There are no additions to the messages tab considered in this tech spec.
+
+Open Questions
+--------------
+
+-   What do we use to canonically track apps we may want to exclude? Is there a bundle ID? The name of app (fragile)? Something else? Do we use different IDs across platforms?
+-   How hard would these checks be to run in Android daemon or PeriodicWorkRequest? (If substantially harder, Santiago open to doing it in Android app - which reduces how often it runs, as the app must be open in order for it to run.) On desktop platforms, the app is always active when the VPN is active. This is not true on mobile, which is why this is considered separately.
+-   Any easy ways to categorize games? (See [VPN-4433](https://mozilla-hub.atlassian.net/browse/VPN-4433).)
+-   How will we decide which apps are initially on the suggested list?
+-   Does the suggested list just include specific apps, or can it include entire directories (like well-known Windows game directories) and/or allow wildcard matching?
+
```

Comment:
- How can we validate apps before adding them to the list to be sure we don't recommend excluding an app with egregious tracking practices?

Reply:
Will be discussed on Monday's meeting